### PR TITLE
update Read state outputs permission description

### PR DESCRIPTION
**Pull request by slavrd**
## Labels

- [x] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

## Description

The `Read state outputs` permission for a Terraform Cloud workspace will grant access to the State Version Outputs API endpoint. 
The `terraform output` and the `terraform_remote_sate` data source do not currently use this endpoint to retrieve the workspace's state outputs. They read the full state and so require the `Read state` permission to work.

It would be good ot clarify that in the documentation as it seems users are getting confused.

### CODE REVIEW

File: content/source/docs/cloud/users-teams-organizations/permissions.html.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ The following workspace permissions can be granted to teams on a per-workspace b
...
+    - **Read state outputs:** — Allows users to access values in the workspace's most recent Terraform state that have been explicitly marked as public outputs. Output values are often used as an interface between separate workspaces that manage loosely coupled collections of infrastructure, so their contents can be relevant to people who have no direct responsibility for the managed infrastructure but still indirectly use some of its functions. This permission is required to access the [State Version Outputs](https://www.terraform.io/docs/cloud/api/state-version-outputs.html) API endpoint.
+ 
+        -> **Note:** To use the `terraform output` command or the `terraform_remote_state` data source against the workspace the **Read state versions** permission is required.
```
Comment (laurapacilio) [inline content/source/docs/cloud/users-teams-organizations/permissions.html.md:80]:
```suggestion
        -> **Note:** **Read state versions** permission is required to use the `terraform output` command or the `terraform_remote_state` data source against the workspace.
```

**Review by laurapacilio — APPROVED** [review]
Left a comment to make the note flow better. Otherwise, I'm assuming this content is correct, and looks good to me!

**Comment by slavrd** [comment]
Thanks for the review and suggestions @laurapacilio. I've committed them.

**Comment by laurapacilio** [comment]
You're welcome!  You're still waiting on a review from @nfagerlund to merge, right?

**Comment by slavrd** [comment]
Honestly, I am not sure. I am not really clear if there are specific people that should approve and merge these PRs or I should do it myself.

**Comment by laurapacilio** [comment]
That's fair - My take is that once something has been approved, you can go ahead and merge if you have the permissions. So, feel free to merge here whenever you feel comfortable - thank you!

**Pull request merged**